packages/internal/src/storage-test-utils.js

```diff
@@ -254,3 +255,43 @@ export const makeMockChainStorageRoot = () => {
     keys: () => [...data.keys()],
   });
 };
+
+/**
+ * @param {import('ava').ExecutionContext<unknown>} t
+ * @param {MockChainStorageRoot | FakeStorageKit} storage
+ * @param {({ note: string } | { node: string; owner: string }) &
+ *   ({ pattern: string; replacement: string } | {})} opts
+ */
+export const documentStorageSchema = async (t, storage, opts) => {
```

Comment:
internal... hm... we do want it exported from the SDK. I guess governance can continue to play that role.